src/main/java/com/cloudbees/plugins/deployer/DeployEvent.java

```diff
@@ -28,15 +28,14 @@
 import hudson.model.AbstractBuild;
 import hudson.model.Cause;
 
-import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 /**
  * @author stephenc
  */
-public class DeployEvent implements Serializable {
+public class DeployEvent {
```

Comment:
I will leave it to @christ66, @stephenc and @olamy since they are the plugin maintainers. There is a way to make the class serializable, but I am not sure it worth effort (though I would really vote for it if I was the maintainer)